### PR TITLE
Adding records-management page, fixing #35

### LINKED ISSUE #35: Document guidance for records management identity wrt OSS
### Description of Problem:

In #26 "how do banks deal with records management identity wrt OSS" came up as a common problem.

### Potential Solutions:

We should provide a new resource page for this topic on the [OSR microsite](https://osr.finos.org).

### PR DESCRIPTION
**Pull request by psmulovics**
Closes #35

THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE FINOS CORPORATE CONTRIBUTOR LICENSE AGREEMENT.

THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

File: docs/operations/records-management.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+id: records-management
+title: Considerations and Guidance for Records Management regards Open Source
+sidebar_label: Record Management Policies
+---
+
+Financial companies do define their retention policies when comes to record management, like how email communication, internal source code, internal chat systems (all involving possible counter parties) are applied to all firm business; some of these even surveiled in real time. However, such policies only do apply to what is considered firm business. As by definiton the Open Source work is not considered a firm business, only the firm wide communication policies that would be applied to the incoming email notifications from GitHub, would be applicable here.
```
Comment (vmbrasseur) [inline docs/operations/records-management.md:7]: * Remove the 'do' here.
* Link "open source work is not considered a firm business" to the Electronic Communications Policies resource page
* Add another paragraph advising people to double check the policies with their lawyers/etc.
* Change to read "Open source work is not considered firm-related business"

**Review by vmbrasseur — CHANGES_REQUESTED** [review]
Per comments in the July 6th OSR call.

File: docs/operations/records-management.md (hunk 1 above)
Comment (psmulovics) [reply docs/operations/records-management.md]: @vmbrasseur please recheck; I think I covered everything from the meeting

**Comment by vmbrasseur** [comment]
lgtm!

**Pull request merged**